src/libs/fsdrv/lv_fsdrv.h

```diff
@@ -43,6 +43,10 @@ void lv_fs_posix_init(void);
 void lv_fs_win32_init(void);
 #endif
 
+#if LV_USE_FS_MEMFS != '\0'
```

Comment:
Actually it's mistake with the other drivers as well. It should be:

```suggestion
#if LV_USE_FS_MEMFS
```

Reply:
Shall I fix it, or do you want to fix it for all drivers?

Reply:
I've fixed it.